sdk/evm/__tests__/versions.test.ts

```diff
@@ -0,0 +1,44 @@
+import { Wormhole } from "@wormhole-foundation/sdk-connect";
+import { EvmPlatform } from "@wormhole-foundation/sdk-evm";
+import { EvmNtt } from "../src/index.js";
+
+const wh = new Wormhole("Testnet", [EvmPlatform]);
+
+const overrides = {
+  Avalanche: {
+    token: "0x72CAaa7e9889E0a63e016748179b43911A3ec9e5",
+    manager: "0x22D00F8aCcC2da440c937104BA49AfD8261a660F",
+    transceiver: {
+      wormhole: "0xeA8D34fa9147863e486d2d07AB92b8218CF58C0E",
+    },
+  },
+};
+
+describe("ABI Versions Test", function () {
```

Comment:
Solana now has a version (example of how to fetch is in this PR https://github.com/wormhole-foundation/example-native-token-transfers/pull/400). Should we add a version check to the Ntt Solana implementation?

Reply:
added in https://github.com/wormhole-foundation/example-native-token-transfers/pull/407